### ISSUE TITLE
Can not choose stealth option when sending

### ISSUE THREAD

**Issue by Pafke01** [vergecurrency/VERGE#436]
# Issue Template

## Context
Running on Windows 8 (64) I would like to sent a amount in Stealth mode.
But I can only choose Sent ...How can I choose sent in Stealth mode ?

By the way the top menu items  are in Dutch ....But it is an English Windows version.

## User Story:

* "I'm submitting a ... "
  - [ ] bug report
  - [ ] feature request
  - [X ] support request 
  - [ ] translation request 

## Screenshots
![sent](https://user-images.githubusercontent.com/35295279/34770008-e7c13682-f5ff-11e7-87fa-cec3aacb8c28.jpg)

## Steps to Reproduce (for bugs)

* Steps :
1.
2.
3.
4.

## Your Environment

* Verge Release: 4.01

* CPU: Intel i3
* GPU:
* RAM: 16 Gig
* OS: Windows 8.0
    - [ ] I'm using Mac
    - [ ] I'm using Linux
    - [ ] I'm using Mac

**Comment by techpoy** [comment]
@Pafke01 Friend this is not an issue

1. You don't need to have stealth option for sending coins. you talk to the receiver first and he gives you a stealth address to which you can send the coins in stealth mode. so the address itself is stealth. similarly if you want to receive funds in stealth mode, you create a receiving address with stealth option checked and it would give you a stealth address and you can give this address to your senders to receive money in stealth mode

2. Language is an option. go to Settings -> Options ->Display tab and select your desired user interface language.

I hope this helps.

these two are not issues so kindly close this issue.

**Comment by Pafke01** [comment]
I found it thanks... seems we need a better manual/explanation...;-)

About the Language option : it is scrambled...no way I can choose a language.
![untitled](https://user-images.githubusercontent.com/35295279/34771174-f290d3de-f603-11e7-8e81-2248ecea4f86.jpg)

**Comment by ralphMevision** [comment]
The language is linked to you Windows OS, if you change it to UK or US then the language will change in the core wallet. This is a workaround for now but I guess the dev team is working on a quick fix for this one.

**Comment by Pafke01** [comment]
@ralphMevision 
Basic screen is devided in 2 languages...
![lang3](https://user-images.githubusercontent.com/35295279/34773128-dcc0155e-f60a-11e7-9fc2-4655e729167a.jpg)

OS language is English:

![lang1](https://user-images.githubusercontent.com/35295279/34773148-ee647106-f60a-11e7-9bfa-0c10e11501f6.jpg)

But the region is different....:

![lang2](https://user-images.githubusercontent.com/35295279/34773161-014ee814-f60b-11e7-96c0-e1dd35eb6c38.jpg)

Which one is used...;-)

**Comment by justinvforvendetta** [comment]
ok so the language dropdown needs to be fixed

**Comment by ralphMevision** [comment]
@Pafke01 try to switch your location to UK in the location tab and then restart the core wallet.

**Comment by Pafke01** [comment]
@ralphMevision & @justinvforvendetta 

Found it: seems the language is related to Region-> Format.
After changing it to English (United States) the screen is correct.
I tried the same on a Windows 10 system. Same result.
But due to the cosmetic issue ... it is impossible to change it other then English (default).
The language check should be done on Display Language instead of RegionFormat.

![untitled](https://user-images.githubusercontent.com/35295279/34787599-15b25232-f638-11e7-9daf-57980108e065.jpg)

![untitled1](https://user-images.githubusercontent.com/35295279/34787947-53885768-f639-11e7-86d9-629ea67fc871.jpg)


I am willing to look at it but just trying to understand the github source control system/compilers etc..
I have some experience with C#,php,C etc..
